pihole-google.txt

```diff
@@ -7619,4 +7619,67 @@ www.tofu.jibemobile.com
 www.tokyo.jibemobile.com
 www.viper.jibemobile.com
 www.watermelon.jibemobile.com
-yellowstone.jibemobile.com
\ No newline at end of file
+yellowstone.jibemobile.com
+# MY SUGGESTIONS
+i0.ytimg.com
+i5.ytimg.com
+youtube-noscript.com
+www.youtube-noscript.com
+goog.le
+domains.googlesyndication.com
+appliedsemantics.com
+ads.googlesyndication.com
+.google-analytics.com
+.googletagservices.com
+.doubleclick.com
+.doubleclick.net
+clients9.google.com
+clients0.google.com
+sassets.vevo.com
```

Comment:
```suggestion
```
vevo.com is not a Google domain, see https://whois.domaintools.com/vevo.com

Reply:
Done